projects/step-frontend/src/lib/modules/_common/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,63 @@
+<input id="collapsable-sidebar-toggle-checkbox" type="checkbox" />
+<div class="sidebar">
+  <div id="sidebar-logo-and-burgermenu">
+    <span class="visible-when-collapsed">
+      <label for="collapsable-sidebar-toggle-checkbox" (click)="toggleOpenClose()"
+        ><i class="glyphicon glyphicon-menu-hamburger"></i></label
+    ></span>
+    <span class="sidebar-aligned-text"><img class="sidebar-logo" alt="" [src]="logo" /></span>
+  </div>
+  <div id="sidebar-home-and-settings">
+    <span class="visible-when-collapsed sidebar-home-button" (click)="navigateTo('home')"
+      ><i class="glyphicon glyphicon-home"></i
+    ></span>
+    <span class="sidebar-home-and-settings-divider">|</span>
+    <span (click)="navigateTo('admin')" *ngIf="_authService.hasRight('admin')" class="sidebar-home-and-settings-icon">
+      <i class="glyphicon glyphicon-cog"></i
+    ></span>
+  </div>
+  <div id="sidebar-categories">
+    <div id="sidebar-categories-tabs">
+      <!-- MENU -->
+      <div
+        class="sidebar-collapsable-tab"
+        *ngFor="let mainMenuEntry of _viewRegistryService.registeredMenuEntries | menuFilter: 'root'; index as i"
+        [attr.lastSidebarMainMenuLabel]="i === mainMenuEntriesAmount"
+      >
+        <input
+          #mainMenuCheckBox
+          [name]="mainMenuEntry.viewId"
+          id="collapsable-tab-{{ mainMenuEntry.label }}"
+          type="checkbox"
+        />
+        <label for="collapsable-tab-{{ mainMenuEntry.label }}">
+          <span [class]="mainMenuEntry.icon + ' visible-when-collapsed'"></span>
```

Comment:
Instead of concatenation in template the better approach would be
```
<span class="visible-when-collapsed" [ngClass]="mainMenuEntry.icon"></span>
```

Reply:
Thanks for the feedback! All done!